.github/workflows/basic.yml

```diff
@@ -39,8 +39,8 @@ jobs:
         uses: actions/cache@v3
         with:
           path: "**/node_modules"
-          key: ${{ runner.os }}-cacheNodeModules22-${{ steps.nodeModulesCacheKey.outputs.value }}
-          restore-keys: ${{ runner.os }}-cacheNodeModules22-
+          key: ${{ runner.os }}-cacheNodeModules23-${{ steps.nodeModulesCacheKey.outputs.value }}
```

Comment:
If we continue to change this for each electron version then maybe this suffix should be more specific and have the electron major version in it?

Reply:
Yeah makes sense, will update the keys.

Reply:
Investigated a bit more before changing the key name as to why the computed key from https://github.com/microsoft/vscode/blob/main/build/azure-pipelines/common/computeNodeModulesCacheKey.ts is not sufficient in Github CI whereas it works well for azure pipelines when just updating the `.cachesalt`. Turns out we are using `restore-keys` option which can use stale cache when no cache hit was found https://github.com/actions/cache#inputs, I just removed it which should avoid bumping the key name with every Electron update.